### PR TITLE
NEWS: Prepare for 0.10 release

### PR DESCRIPTION
**Pull request by tgraf**
Signed-off-by: Thomas Graf <thomas@cilium.io>

### CODE REVIEW

**Review by aalemayhu — APPROVED** [review]

**Review by borkmann — APPROVED** [review]
Looks good! #969 could probably still go in there.

**Review by amreshakim — APPROVED** [review]
LGTM!

File: NEWS.rst [hunk 1]
```diff
@@ -2,9 +2,112 @@
...
+Kubernetes
+----------
+* Drop support for extensions/v1beta1/NetworkPolicy and support
+  networking.k8s.io/v1/NetworkPolicy (1150_)
+* Allow fine grained inter namespace policy control. It is now possible to
+  specify policy rules which allow individual pods from another namespace to
+  access a pod (1103_)
+* The CiliumNetworkPolicy ThirdPartyResource now supports carrying a list of
+  rules to update atomically (1055_)
+* The example DaemonSet now scheduled Cilium pods onto nodes which are not
```
Comment (ashwinp) [inline NEWS.rst]: now scheduled -> now schedules

File: NEWS.rst [hunk 2]
```diff
@@ -2,9 +2,112 @@
...
+  exists (871_)
+* Full RBAC support (873_, 875_)
+* Correctly implement ClusterIP portion of k8s service types LoadBalancer and
+  NodePort (1098_)
+* The cilium and consul pod in the example DaemonSet now have health checks
+  (925_, 938_)
+* Correctly ignore headless services without a warning in the log (932_)
+* Derive node-name automatically (1090_)
+* Labels are now attached to endpoints instead of containers. This will allow
+  to support labels attached to thing other than containers (1121_)
```
Comment (ashwinp) [inline NEWS.rst]: typo: things

**Review by ashwinp — APPROVED** [review]

**Comment by tgraf** [comment]
> Looks good! #969 could probably still go in there.

Good point, will add

**Pull request merged**